pkgs/tools/audio/openai-whisper-cpp/default.nix

```diff
@@ -42,9 +43,9 @@ effectiveStdenv.mkDerivation (finalAttrs: {
   nativeBuildInputs = [
       which
       makeWrapper
-    ] ++ lib.optionals cudaSupport ( with cudaPackages ;[
+    ] ++ lib.optionals cudaSupport ((with cudaPackages ;[
       cuda_nvcc
-
+    ]) ++ [
```

Comment:
The new indents aren't stable under any existing autoformatters

Reply:
removed the with syntax